src/controller/tag/controller.go

```
@@ -116,7 +118,22 @@ func (c *controller) Ensure(ctx context.Context, repositoryID, artifactID int64,
 		tag.ArtifactID = artifactID
 		tag.Name = name
 		tag.PushTime = time.Now()
+
 		tagID, err = c.Create(ctx, tag)
```

err is being ignored here